front/src/gc-components/GCGridCol.tsx

```diff
@@ -15,7 +15,7 @@ interface GCGridColProps extends GridProps{
 
 export const GCGridCol = ({ className, children, span, xs, sm, md, lg, xl, ...props }: GCGridColProps) => {
     return (
-        <Grid.Col className= {className} span={span} xs={xs} sm={sm} md={md} lg={lg} xl={xl} {...props}>
+        <Grid.Col className={className} span={span} xs={xs} sm={sm} md={md} lg={lg} xl={xl} {...props}>
```

Comment:
I think all that changed here is just code style @Logovas1994 plz control that prettier works well, and this is write code style